src/datatablerow/datatablerow.jl

```diff
@@ -62,32 +58,19 @@ end
 Base.hash(r::DataTableRow, h::UInt = zero(UInt)) = rowhash(r.dt, r.row, h)
 
 # comparison of DataTable rows
+# only the rows of the same DataTable could be compared
 # rows are equal if they have the same values (while the row indices could differ)
-# returns Nullable{Bool}
-# if all non-null values are equal, but there are nulls, returns null
-function @compat(Base.:(==))(r1::DataTableRow, r2::DataTableRow)
```

Comment:
Let's keep this method for now too, unless it's an issue for this PR, as the debate about what `==` should do with nullables is a complex one.